codedigger/atcoder/serializers.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework import serializers
+from problem.models import Problem, atcoder_contest
+
+
+class AtcoderProblemSerializer(serializers.ModelSerializer):
```

Comment:
If this is the exact same copy from Problem App. Make sure to remove from there and import this function wherever we are using AtcoderProblemSerializer.